lib/web_ui/lib/src/engine/canvaskit/surface_factory.dart

```diff
@@ -29,6 +29,12 @@ class SurfaceFactory {
   /// The maximum number of surfaces which can be live at once.
   final int maximumSurfaces;
 
+  /// The maximum number of assignable overlays.
+  ///
+  /// This is just `maximumSurfaces - 2` (the maximum number of surfaces minus
+  /// the base surface and overlay surface).
```

Comment:
nit: did you mean base surface and _backup_ surface?

Reply:
Yes, done